stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import matplotlib.pyplot as plt
</pre><p>Jennifer Ding:</p><p>Since visualisation is a key goal of the piece, it would be nice to explain the use of both matplotlib and seaborn which have different strengths/weaknesses, or at least mention seaborn as well in Research Agenda item <a href="https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Falan-turing-institute%2FTuringDataStories%2Fissues%2F1&amp;data=05%7C01%7C%7C4b85c45ca1ca47c7664408da753c5082%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637951199152866481%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&amp;sdata=BkgeQexSEzDIN9PTud8GOp6wUPDdJ0AfZOHq3SRDq4g%3D&amp;reserved=0" target="_blank">#1</a></p><p> </p><p>Joe Cerniglia:</p><p>That is an excellent idea.&nbsp;I will do that.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>